doc/api/tty.md

```diff
@@ -50,6 +50,14 @@ raw device. Defaults to `false`.
 added: v0.7.7
 -->
 
+Allows configuration of `tty.ReadStream` so that it operates as a raw device.
+
+For `stdin`, this mean that input is always available character-by-character
```

Comment:
Well stdout doesn't usually receive this kind of input? Honestly I'm not really sure what this does to stdout...